### ISSUE TITLE
SQL entry

### ISSUE THREAD

**Issue by nicholasjhorton** [hglanz/DS-Blog-Ex#37]

**Comment by nicholasjhorton** [comment]
I've drafted this: comments and suggestions welcomed by @hardin47 and @hglanz.  I worry that I've done something wrong and missed a step with the final figure.

**Comment by hardin47** [comment]
REALLY GREAT SQL blog!!!  I already learned a ton, and will use it in my class in the fall.  That said, I have a few comments, likely due to me being a SQL total newbie.

I made a few edits here:  https://github.com/hglanz/DS-Blog-Ex/commit/eb0a76e0161d0d5ad9a1600bf4378a1974b4a93d

* I don't know what this phrase means:  "Next a series of SQL queries can be sent to the database: these return R dataframes."    [I changed it to: "Next a series of SQL queries can be sent to the database: each query returns an R dataframe."  but I'm pretty sure that what I wrote is incorrect.]

    - To which line of code are you referring?  Which things are "queries"?  Is EXPLAIN a query?  does it return a dataframe?

    - And do they all return dataframes?  Which things return dataframes?  Should we try to define dataframe vs database vs table?

* What is a " `dplyr` table object" ?  Is that really a thing?

* Not sure we should list people's email addresses, even if they do exist in a public database.  Select that column out?

* Were you planning to add comments to the URLs in "Learn More" ?  I think they are fine without, but you had left an open parenthesis which indicated you might have more to say.
